From 3faefb4b3dfc074547323058a16f4d92fff431d0 Mon Sep 17 00:00:00 2001
From: Wei Yongjun <weiyongjun1@huawei.com>
Date: Tue, 25 Apr 2017 06:34:46 +0000
Subject: [PATCH 1828/2241] mmc: sdhci-xenon: Remove redundant dev_err call in
 get_dt_pad_ctrl_data()

There is a error message within devm_ioremap_resource
already, so remove the dev_err call to avoid redundant
error message.

Change-Id: I98ac1a165a4ef2905b2dd3faa274689ebad5f65c
Signed-off-by: Wei Yongjun <weiyongjun1@huawei.com>
Signed-off-by: Ulf Hansson <ulf.hansson@linaro.org>
Reviewed-on: http://vgitil04.il.marvell.com:8080/42054
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 drivers/mmc/host/sdhci-xenon-phy.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/mmc/host/sdhci-xenon-phy.c b/drivers/mmc/host/sdhci-xenon-phy.c
index 93ee5ad..e04ceaa 100644
--- a/drivers/mmc/host/sdhci-xenon-phy.c
+++ b/drivers/mmc/host/sdhci-xenon-phy.c
@@ -644,11 +644,8 @@ static int get_dt_pad_ctrl_data(struct sdhci_host *host,
 
 	params->pad_ctrl.reg = devm_ioremap_resource(mmc_dev(host->mmc),
 						     &iomem);
-	if (IS_ERR(params->pad_ctrl.reg)) {
-		dev_err(mmc_dev(host->mmc), "Unable to get SoC PHY PAD ctrl register for %s\n",
-			np->name);
+	if (IS_ERR(params->pad_ctrl.reg))
 		return PTR_ERR(params->pad_ctrl.reg);
-	}
 
 	ret = of_property_read_string(np, "marvell,pad-type", &name);
 	if (ret) {
-- 
2.7.4

